packages/itertools/_test.pony

```diff
@@ -433,6 +444,16 @@ class iso _TestIterTake is UnitTest
         end)
 
     h.assert_eq[USize](3, infinite.take(3).collect(Array[U64]).size())
+
+    let short = Iter[U64](
+      object is Iterator[U64]
+        var counter: U64 = 0
+        fun ref has_next(): Bool => false
+        fun ref next(): U64^ => counter = counter + 1
+      end
+    )
```

Comment:
The `)` should be after the `end` keyword, like `end)`, rather than on a new line.

Reply:
my bad. this is fixed.